routing/routes/routes.yml

```diff
@@ -0,0 +1,7 @@
+authtwitter-linkback:
+    path:       /linkback
+    defaults:   { _controller: 'SimpleSAML\Module\authtwitter\Controller\Twitter::linkback' }
+
+authtwitter-linkback-legacy:
```

Comment:
This is for existing client registrations I think?

Reply:
Yes, I don't intend to keep to much BC, but I think this is reasonable.. The SSP maintainer may not have access to the Twitter developer pages to change the callback url.

Reply:
So makes sense to update docs to the new preferred url.
And at the same time remove broken images from it.
And at the same time add the config options to it?

Reply:
It's actually impossible to support the old route, because we tell twitter what the callback url is in the request. I therefore removed it.
Docs have been updated with an example configuration, the images were removed